tcf_website/templates/search/search.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base/base.html" %}
+{% load static %}
+
+{% block styles %}
+<link rel="stylesheet" href="{% static 'search/search.css' %}" />
+{% endblock %}
+
+{% block content %}
```

Comment:
Maybe wrap the content block with a `<div class="search">` that way css selectors can be more specific, e.g. `.search hr`

Reply:
fixed